js/initcontext.go

```diff
@@ -69,6 +69,8 @@ type InitContext struct {
 	compatibilityMode lib.CompatibilityMode
 
 	logger logrus.FieldLogger
+
+	sharedObjects *common.SharedObjects
```

Comment:
I don't think this needs to be here. You should be able to just create it via `common.NewSharedObjects()` in `js/bundle.go`, in the `initenv := &common.InitEnvironment{...}`. We need to eventually remove the needlessly special `InitContext` struct anyway, no need to add things to it that it doesn't need.

Reply:
Ah, nevermind, ignore me, that would create a copy per VU... :disappointed: